core/lib/constants/src/contracts.rs

```diff
@@ -137,3 +137,11 @@ pub const MINT_AND_BURN_ADDRESS: H160 = H160::zero();
 
 // The `storage_log.value` database value for a contract that was deployed in a failed transaction.
 pub const FAILED_CONTRACT_DEPLOYMENT_BYTECODE_HASH: H256 = H256::zero();
+
+pub const ETHEREUM_SHARED_BRIDGE_ADDRESS: Address = H160([
```

Comment:
I'd suggest change the name, I read it like "the address of the shared bridge on ethereum". Probably better if it was something like `SHARED_BRIDGE_ETHER_TOKEN_ADDRESS`